apps/heureka/src/components/common/Breadcrumb/Breadcrumb.tsx

```diff
@@ -23,17 +23,15 @@ export const Breadcrumb = () => {
   }, [])
 
   return (
-    <Container py px={false}>
-      <BreadcrumbContainer>
-        <BreadcrumbItem icon="home" label="Services" onClick={() => handleClick(UserView.Services)} />
-        {selectedView.viewId === UserView.ServiceDetails && (
-          <>
-            <BreadcrumbItem
-              label={capitalizeFirstLetter((selectedView.params as ServiceDetailViewParams).service.name)}
-            />
-          </>
-        )}
-      </BreadcrumbContainer>
-    </Container>
+    <BreadcrumbContainer className="mb-6">
+      <BreadcrumbItem icon="home" label="Services" onClick={() => handleClick(UserView.Services)} />
+      {selectedView.viewId === UserView.ServiceDetails && (
+        <>
```

Comment:
The `Fragment` isn't really necessary here. You can simplify this and remove the wrapping `Fragment`. I'm guessing it's a leftover from an earlier state where it was necessary as a wrapper for multiple sibling elements.

Reply:
Yes, you are right!